common/hash_data_adapter_hidden_files.go

```diff
@@ -83,9 +93,8 @@ func (a *HiddenFileDataAdapter) SetHashData(relativePath string, data *SyncHashD
 
 	// Push types around to check for OS-specific hide file method
 	if adapter, canHide := any(a).(interface{ HideFile(string) error }); canHide {
-		dataFile := a.getDataPath(relativePath)
-
-		err := adapter.HideFile(dataFile)
+		metaFile := a.getHashPath(relativePath)
```

Comment:
We shall put some comments here on why we are changing this code and why meta shall be hidden

Reply:
added